src/fetch/headers.js

```diff
@@ -118,11 +118,9 @@ class Headers {
     }
   }
 
-  * keys() {
-    // return Object.keys(this[INTERNALS].map).sort();
-    for (const name of Object.keys(this[INTERNALS].map).sort()) {
-      yield name;
-    }
+  keys() {
```

Comment:
why not just `return this[INTERNALS].map.keys()` ?

Reply:
`Map` is insertion ordered while the iterator needs to return the keys in alphabetical order.

Reply:
> `Map` is insertion ordered while the iterator needs to return the keys in alphabetical order.

why ?